pre_push.py

```diff
@@ -1,9 +1,14 @@
 #!/usr/bin/env python3
 """Run static analysis on the project."""
+
+import sys
+
```

Comment:
There needn't be this blank line here. Also argparse should be just before `import sys`.